tests/requests/test_samples_declarations.py

```diff
@@ -0,0 +1,251 @@
+    assert response.status_code == HTTPStatus.UNPROCESSABLE_ENTITY, response.json
+    assert len(response.json["_items"]) == 2
+    assert response.json["_status"] == "ERR"
+    assert response.json["_items"][0]["_status"] == "ERR"
+    assert response.json["_items"][0]["_issues"] == {
+        "value_in_sequencing": "unallowed value wrong answer!!"
+    }
+    assert response.json["_items"][1]["_status"] == "OK"
+
+
+def test_wrong_value_for_declared_at(client, samples, samples_declarations):
+    response = post_authorized_create_samples_declaration(
+        client,
+        [
+            {
+                "root_sample_id": "MCM001",
+                "value_in_sequencing": "Unknown",
+                "declared_at": TIMESTAMP,
+            },
+            {
+                "root_sample_id": "MCM003",
+                "value_in_sequencing": "Yes",
+                "declared_at": "wrong time mate!!",
+            },
+        ],
+    )
+    assert response.status_code == HTTPStatus.UNPROCESSABLE_ENTITY, response.json
+    assert len(response.json["_items"]) == 2
+    assert response.json["_status"] == "ERR"
+    assert response.json["_items"][0]["_status"] == "OK"
+    assert response.json["_items"][1]["_status"] == "ERR"
+    assert response.json["_items"][1]["_issues"] == {"declared_at": "must be of datetime type"}
+
+
+def test_wrong_value_for_root_sample_id(client, samples, samples_declarations):
+    response = post_authorized_create_samples_declaration(
+        client,
+        [
+            {"root_sample_id": True, "value_in_sequencing": "Unknown", "declared_at": TIMESTAMP,},
+            {"root_sample_id": "MCM003", "value_in_sequencing": "Yes", "declared_at": TIMESTAMP,},
+        ],
+    )
+    assert response.status_code == HTTPStatus.UNPROCESSABLE_ENTITY, response.json
+    assert len(response.json["_items"]) == 2
+    assert response.json["_status"] == "ERR"
+    assert response.json["_items"][0]["_status"] == "ERR"
+    assert response.json["_items"][0]["_issues"] == {"root_sample_id": "must be of string type"}
+    assert response.json["_items"][1]["_status"] == "OK"
+
+
+def test_validate_sample_exist_in_samples_table(client, samples, samples_declarations):
+    response = post_authorized_create_samples_declaration(
+        client,
+        [
+            {
+                "root_sample_id": "MCM001",
+                "value_in_sequencing": "Unknown",
+                "declared_at": TIMESTAMP,
+            },
+            {
+                "root_sample_id": "MCM_WRONG_VALUE",
+                "value_in_sequencing": "Yes",
+                "declared_at": TIMESTAMP,
+            },
+        ],
+    )
+    assert response.status_code == HTTPStatus.UNPROCESSABLE_ENTITY, response.json
+    assert len(response.json["_items"]) == 2
+    assert response.json["_status"] == "ERR"
+    assert response.json["_items"][0]["_status"] == "OK"
+    assert response.json["_items"][1]["_status"] == "ERR"
+    # TODO: check error message
```

Comment:
Just this todo, adding item, error is our none existing samples message

Reply:
Added